automation/test-execution/ansible/tests/test_repeatability.py

```
@@ -0,0 +1,429 @@
+            except AssertionError as e:
+                print(f"  ✗ {method_name}: {e}")
+                failed_tests.append((test_class.__name__, method_name, str(e)))
+            except Exception as e:
+                print(f"  ✗ {method_name}: Unexpected error: {e}")
+                failed_tests.append((test_class.__name__, method_name, str(e)))
+
+    print(f"\n{'='*60}")
+    print(f"Results: {passed_tests}/{total_tests} tests passed")
+
+    if failed_tests:
+        print(f"\nFailed tests:")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)